lib/firmwares.json

```diff
@@ -24,7 +24,8 @@
       "repo": "url",
       "npm": {
           "package": "node-pixel",
-          "version": "^0.4.1"
+          "repo": "git+https://github.com/ajfisher/node-pixel",
```

Comment:
@stevemao this is a slight artefact, if you can update to use the latest version from an npm install form that would be good as now we have a stable version.